docs/change-log.md

```diff
@@ -18,6 +18,7 @@
 
 - API: `m.redraw()` is always asynchronous ([#1592](https://github.com/MithrilJS/mithril.js/pull/1592))
 - API: `m.mount()` will only render its own root when called, it will not trigger a `redraw()` ([#1592](https://github.com/MithrilJS/mithril.js/pull/1592))
+- API: `vnode.state = ...` is no longer supported. Instead, an error is thrown if `vnode.state` changes upon the invocation of a lifecycle hook.
```

Comment:
I don't think this is clear enough. Something like 

> Assigning a new object to `vnode.state` (`vnode.state = ...`) is no longer supported

would help me understand.

Reply:
Fixed in a force push. 👍